application/src/main/java/com/github/pieter_groenendijk/model/Copies.java

```diff
@@ -0,0 +1,65 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+
+@Entity
+@Table (name = "Copies")
+public class Copies {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long copyId;
+
+    @Column (name = "numberOfCopies", nullable = false)
+    private long numberOfCopies;
+   @ManyToOne
+    @JoinColumn(name = "productId", nullable = false)
+    private PhysicalProduct physicalProduct;
+
+    @Column (name = "availabilityStatus", nullable = false, length = 50)
+    private String availabilityStatus;
+
+    @Column (name = "isAvailable", nullable = false)
+    private boolean isAvailable;
```

Comment:
Wat is het verschil tussen `availabilityStatus` en `isAvailable`? Eentje lijkt mij genoeg, of?